### DISCUSSION TITLE
理解 Kotlin 的 suspend 函数

### DISCUSSION THREAD

**Discussion by yujinyan** [yujinyan/blog#177] [Comments]
https://blog.yujinyan.me/posts/understanding-kotlin-suspend-functions/

**Comment by yujinyan** [comment]
接下来想写篇之前挺困惑的 `CoroutineContext` 和 `CoroutineScope` 🤔

**Comment by yujinyan** [comment]
看到这篇用 `supend` 做深递归的例子，感觉非常适合加进来介绍😁 
https://elizarov.medium.com/deep-recursion-with-coroutines-7c53e15993e3

**Comment by yujinyan** [comment]
今天发现用这个回旋镖（boomerang）来表示挂起点感觉更好一点，但是这个 emoji 🪃 太新了，Android / Windows 还不支持

**Comment by Dawnman** [comment]
协的好棒。
表示 suspend 到 emoji 确实可以帮助理解代码。

**Comment by est7** [comment]
写的很棒,老师多多更新,可能是工作忙了都没见这两年的更新了

**Comment by JarvisBuop** [comment]
学习了，对suspend理解更深了
